database_config/init.sql

```diff
@@ -0,0 +1,178 @@
+9   Software Dev QA Co-op   http://careers2.hiredesk.net/viewjobs/JobDetail.asp?Comp=Fortinet&sPERS_ID=&TP_ID=1&JB_ID=&PROJ_ID=%7BB1F386BD-9EB9-4E83-9123-CBCF711E3175%7D&LAN=en-US&BackUrl=viewjobs/Default.asp&ccuid=15792325325  Fortinet is looking for a Software Dev QA Engineer to join our UI (User interface) Development team to focus on GUI testing. As a team member, you would be responsible for testing the FortiOS GUI for managing FortiGate firewall appliances. You will be working closely with the interface development team to verify new features, automate test cases and improve interface usability and performance.\nThis is an exciting opportunity to work with the latest in web application and networking technologies, such as Security Fabric, Next Generation Firewall, UTM security, cloud application control, VPN management, WiFi network topologies, and firewall monitoring.\n\nJob Responsibilities:\n\nExecute feature and bug fix testing for FortiOS GUI application using a combination of manual and automated\ntesting techniques.\nIdentify, document, and track software defects found during testing.\nAutomate front-end testing using automation tools such as Appium, Selenium.\nImprove testing and development processes using various automation techniques.\nWork closely with developers to ensure defects are correctly identified and fixed.\n\nJob Skills Required:\n\nSoftware testing experience including browser testing\nDemonstrated ability to logically and analytically troubleshoot mobile/web applications\nKnowledge of general QA procedures and methodologies, as well as software development fundamentals\nBasic networking knowledge (IP, NAT, Firewall, Routing)\nDemonstrated ability to write clear and reproducible problem reports, and test results\nExperience with automation testing tools is an asset\nExperience in code review systems and bug tracking services is an asset\nExperience working in a Linux environment\nDemonstrated skills in scripting languages such as Python, bash or Perl are considered assets\n\nEducational Requirements:\n\nA degree or technical diploma in Computer Science, Computer Technology, or related field    Burnaby Canada  49.322299999999998  -123.02209999999999 Fortinet    \N  3500    BC
+\.
+
+
+--
+-- Name: job_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
+--
+
+SELECT pg_catalog.setval('public.job_id_seq', 2, true);
```

Comment:
My mistake, I'm not sure why Postgres dumped it like this. The second argument should be 10. Otherwise it will try to insert the id as 3 for default when inserting. So this line should be 
```
SELECT pg_catalog.setval('public.job_id_seq', 10, true);
```

Reply:
Fixed it !